<table align="left">
  <td>
    <a href="https://colab.research.google.com/github/marco-canas/calculo/blob/main/referents/grajales/5_limits_and_continuty/1_limite_en_un_punto/1_limites.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
  </td>
</table>

Hoy dedico mi lectura a mi familia con Yas y Joseph

<img src = 'https://github.com/marco-canas/didactica_ciencia_datos/blob/main/4_images/3_images_of_students/janis_joseph_marco_granizado.jpg?raw=true' width = 600>

### [Video de apoyo a la lectura interactiva y experimental de este cuaderno](https://www.youtube.com/watch?v=n385JEhrC64)

## [Vínculo al programa del curso de matemáticas I:](https://github.com/marco-canas/2_calculo_I/tree/main)

## [Vínculo al programa del curso de matemáticas II](https://github.com/marco-canas/calculo/blob/main/2_programa_math_II.pdf)

Un reconocimiento a mi hermana:

<img src = 'https://github.com/marco-canas/didactica_ciencia_datos/blob/main/4_images/3_images_of_students/yorla_marco_2022_07_20.jpg?raw=true' width = 600> 



# Llamar a lista o tomar asistencia con python o llamar el equipo para la tarea con video y cuaderno Jupyter sobre medición indirecta de grandes alturas con imágenes

In [2]:
# Llamar a lista o tomar asistencia con python 

import pandas as pd # convertir la lista de estudiantes en un dataframe
import numpy as np # elección aleatoria

# Cargar la lista de estudiantes desde la URL
url = 'https://raw.githubusercontent.com/marco-canas/groups_list/main/math1.csv'
estudiantes = pd.read_csv(url) # convierte el csv en una dataframe

# Convertir la columna 'Nombre' a una lista de Python
lista_estudiantes = estudiantes['Nombre'].tolist()

# Función para llamar a un estudiante aleatoriamente y registrar su nota de seguimiento
def llamar_estudiante():
    if lista_estudiantes:
        estudiante_aleatorio = np.random.choice(lista_estudiantes)
        print(f"Llamando a {estudiante_aleatorio}...")
        lista_estudiantes.remove(estudiante_aleatorio)
    else:
        print("Se han llamado a todos los estudiantes del grupo completo.")


In [8]:
llamar_estudiante()

Llamando a VILLADIEGO GALLEGO JORGE ANDRES...


# Modelo general de clase: [aula invertida](https://www3.gobiernodecanarias.org/medusa/ecoescuela/pedagotic/aprendizaje-invertido-flipped-classroom/)


## Fomento de la lectura literaria asociada a Ingeniería agropecuaria, licenciatura en Matemáticas y ciencias económicas


[Daniel Goleman: La inteligencia emocional](https://github.com/marco-canas/inteligencia_emocional/tree/main/1_para_que_son_las_emociones) 


## Título de la clase: Parte 5. Límites y continuidad de funciones que servirán como modelos económicos

Página 280 del texto de [Lina Grajales: Matemáticas I para ciencias económicas.](https://github.com/marco-canas/calculo/blob/main/referents/grajales/GrajalesLina_2015_MatemticasCienciasEconmicas.pdf) 



### Objetivo de aprendizaje (Enunciar, de manera corta sencilla y clara para todos los asistentes)

* Aprender a describir tendencias correlacionadas utilizando **la representación tabular** de las funciones elaboradas a mano y con la librería pandas de python. 
* Aprender a describir tendencias o aproximaciones utilizando **la representación gráfica** de las funciones elaboradas a mano y con la librería plotly de python.
* Aprender a calcular tendencias o límites de funciones, a partir de **procedimientos algebriacos y aplicación de teoremas sobre límtes de funciones**. 
* Que el administrador de empresas aprenda a describir comportamientos financieros con fluidez apoyado por representaciones tabulares, gráficas y algebraicas y de la mano del concepto de límite de funciones. 

## Establecimiento de prerrequisitos conceptuales para alcanzar el objetivo de aprendizaje trazado.   




Antes de adentrarse en el tema de límites de funciones, es importante tener una base sólida de conceptos matemáticos y algebraicos. Aquí tienes una lista de prerrequisitos conceptuales recomendados:



1. **Álgebra básica**: Debes estar familiarizado con operaciones aritméticas, resolución de ecuaciones lineales y cuadráticas, factorización y simplificación de expresiones algebraicas. (Página 123 de Lina Grajales: Matemáticas I para ciencias económicas). 



2. **Funciones**: Comprender qué es una función y cómo se representa, así como el concepto de dominio y rango de una función.



3. **Notación matemática**: Conocer la notación matemática estándar, como el uso de letras para variables y símbolos matemáticos como sumatorias, productos y exponentes.



4. **Geometría**: Tener una comprensión básica de conceptos geométricos, como el plano cartesiano y las coordenadas.



5. **Trigonometría**: Familiarizarte con las funciones trigonométricas (seno, coseno, tangente) y sus propiedades básicas.



6. **Reglas básicas de límites**: Conocer las reglas básicas para calcular límites, como las reglas de límites de suma, resta, multiplicación y división, así como el límite de una constante y el límite de una función polinómica.



7. **Asíntotas**: Comprender el concepto de asíntotas y cómo identificarlas en una función.



8. **Continuidad**: Saber qué significa que una función sea continua en un punto y cómo se define la continuidad en un intervalo.



9. **Derivadas**: Tener una comprensión básica de las derivadas y su relación con los límites, aunque no es un prerrequisito absoluto, ya que los límites pueden ser introducidos antes o después del cálculo de derivadas.



10. **Gráficas de funciones**: Saber cómo representar gráficamente funciones básicas y cómo analizar el comportamiento de la función alrededor de un punto.

Estos prerrequisitos proporcionarán una base sólida para comprender y abordar el tema de límites de funciones. Si es necesario, puedes repasar o consultar recursos adicionales para fortalecer tu conocimiento en estos temas antes de abordar los límites de funciones.
 

# Para el diseño de la clase tenga en cuenta algunos problemas de aprendizaje

Aquí tienes una lista de problemas de aprendizaje frecuentes que los estudiantes pueden enfrentar al estudiar el concepto de límites de funciones:



1. **Entender la definición formal de límite**: Algunos estudiantes pueden tener dificultades para comprender la definición formal de límite, que implica expresiones matemáticas abstractas y símbolos como $\epsilon$ (épsilon) y $\delta$ (delta).



2. **Interpretar el significado geométrico del límite**: Comprender el significado geométrico del límite como la aproximación del comportamiento de una función en un punto puede ser un desafío para algunos estudiantes.



3. **Resolver límites indeterminados**: Al calcular límites de funciones, pueden surgir situaciones de "indeterminación" como 0/0 o ∞/∞, que pueden ser confusas al principio.



4. **Aplicación de reglas de límites**: Aplicar correctamente las reglas de límites para funciones algebraicas, trigonométricas o exponenciales puede ser complicado, especialmente cuando se enfrentan a funciones más complejas.



5. **Diferenciar entre límites laterales y límites en un punto**: Comprender la diferencia entre límites laterales (uno desde el lado izquierdo y otro desde el lado derecho) y el límite en un punto puede ser confuso para algunos estudiantes.



6. **Identificar asíntotas**: Al estudiar límites, es importante identificar las asíntotas verticales y horizontales de una función, lo cual puede ser un desafío.



7. **Resolver límites que requieren técnicas especiales**: Algunos límites pueden requerir técnicas especiales, como la sustitución trigonométrica o el uso de la regla de L'Hôpital, lo cual puede ser difícil de aplicar correctamente.



8. **Uso adecuado de notación límite**: Utilizar la notación de límite correctamente puede ser problemático para algunos estudiantes, especialmente al expresar límites en términos de variables.



9. **Errores aritméticos y algebraicos**: Los estudiantes pueden cometer errores aritméticos o algebraicos al simplificar expresiones mientras calculan límites, lo que puede conducir a respuestas incorrectas.



10. **Relacionar límites con otros conceptos matemáticos**: Algunos estudiantes pueden tener dificultades para relacionar el concepto de límites con otros temas matemáticos, como continuidad o derivadas.



Para superar estos problemas de aprendizaje, es esencial que los estudiantes practiquen con una variedad de ejercicios, busquen la ayuda de sus profesores o tutores cuando sea necesario y consulten recursos adicionales, como libros o tutoriales en línea. La paciencia y la perseverancia son clave para desarrollar una comprensión sólida de los límites de funciones.

## Cuestionario para realizar labores de memorización, observación y síntesis de la información (Además de diagnóstico)



Un cuestionario para establecer un diagnóstico de saberes previos de los estudiantes antes de abordar el tema de límites de funciones. Este cuestionario está diseñado para evaluar la comprensión de conceptos matemáticos fundamentales relacionados con funciones y álgebra, que son prerrequisitos para entender los límites de funciones.



1. ¿Qué es una función en matemáticas?    
a) Un conjunto de números reales.  
b) **Una relación entre dos conjuntos donde cada elemento del primer conjunto se asocia con un solo elemento del segundo conjunto.**  
c) Una operación aritmética.  



2. ¿Cuál es la diferencia entre una función y una ecuación?  
a) Una función contiene variables y una ecuación no.  
b) Una función es una igualdad matemática y una ecuación no.  
c) No hay diferencia, ambos términos se usan indistintamente.  
d) La función es una igualdad que establece una relación, mientras que una ecuación es una igualdad que estable una pregunta a resolver. 



3. ¿Cuál es el dominio de una función?
a) El conjunto de todas las salidas posibles de la función.
b) El conjunto de todas las entradas posibles de la función.
c) El conjunto de todas las respuestas correctas de una función.



4. ¿Qué es una asíntota en una gráfica de función?
a) Un punto donde la función alcanza su máximo o mínimo.
b) Una recta vertical u horizontal que la gráfica de la función se acerca infinitamente pero nunca toca.
c) Un punto donde la gráfica de la función se corta con el eje de coordenadas.



5. ¿Qué representa el límite de una función en un punto dado?  
   a) El valor real de la función en ese punto.  
   b) La tasa de cambio instantáneo de la función en ese punto.  
   c) La aproximación del comportamiento de la función en ese punto.  



6. ¿Qué significa que un límite sea "indeterminado"?  
   a) Que el límite no existe.  
   b) Que el límite es igual a cero.  
   c) Que no se puede determinar el valor del límite usando las reglas estándar.  



7. ¿Cuál de las siguientes reglas de límites es válida?
a) Límite de una suma es la suma de los límites.
b) Límite de un producto es el producto de los límites.
c) Límite de una división es la división de los límites.



8. ¿Cuál es el límite de la función $f(x) = 3x + 5$ cuando $x$ se acerca a $2$?  
a) 3  
b) 7  
c) 11  



9. ¿Cuál es el límite de la función $g(x) = sin(x) / x$ cuando $x$ se acerca a $0$?  
a) 0  
b) 1  
c) No existe límite.  



10. ¿Qué se puede concluir si el límite de una función en un punto dado es igual al valor de la función en ese punto?
a) La función es continua en ese punto.
b) La función tiene una asíntota en ese punto.
c) El límite no existe.

Espero que este cuestionario te ayude a evaluar los saberes previos de los estudiantes antes de abordar el tema de límites de funciones. Puedes utilizar las respuestas para identificar las áreas de mayor dificultad y adaptar tus estrategias de enseñanza para abordar esas necesidades específicas.

## [Video de apoyo a la parte introduccitia de esta clase sobre límites de funciones](https://www.youtube.com/watch?v=QY3QPE-LchY)

## El componente histórico y epistemológico

Aquí tienes algunos componentes históricos y epistemológicos importantes para enseñar el concepto de límite de una función en un punto:

1. **Antecedentes históricos**:
   - El concepto de límite tiene sus raíces en los trabajos de matemáticos como Arquímedes y Eudoxo en la antigua Grecia, quienes utilizaron métodos geométricos para aproximarse al cálculo de áreas y volúmenes, sentando las bases para el concepto de límite.
   - Durante el Renacimiento, los matemáticos como Isaac Barrow, Pierre de Fermat y René Descartes desarrollaron ideas que contribuyeron al concepto de límite y al cálculo diferencial.
   - El trabajo de Isaac Newton y Gottfried Wilhelm Leibniz en el siglo XVII fue fundamental en el desarrollo del cálculo y en la formalización del concepto de límite.



2. **Desarrollo epistemológico**:
   - El concepto de límite se desarrolla para abordar las limitaciones de los métodos algebraicos y geométricos en el cálculo de ciertas cantidades, como tasas de cambio instantáneo y áreas bajo curvas.
   - El enfoque epsilon-delta introducido por Karl Weierstrass en el siglo XIX proporcionó una base rigurosa para el estudio de los límites y ayudó a resolver problemas de convergencia y continuidad en el análisis matemático.
   - La comprensión del límite implica la idea de acercamiento infinitesimal, donde se estudia el comportamiento de una función a medida que la variable independiente se aproxima a un valor específico.
   - El concepto de límite es esencial en el cálculo y se aplica en diversas áreas de las matemáticas y las ciencias, desde la física hasta la economía y la ingeniería.



Al integrar estos aspectos históricos y epistemológicos en la enseñanza del concepto de límite de una función en un punto, los estudiantes pueden comprender mejor su importancia, desarrollo y aplicación en diferentes contextos matemáticos y científicos.

## Secuencia conceptual introductoria a la clase

In [2]:
from graphviz import Digraph

# Crear un nuevo grafo dirigido
dot = Digraph()

# Añadir nodos y relaciones al grafo
main_concept = 'Límites de funciones'
dot.node('A', main_concept)
dot.node('B', 'Aproximación')
dot.node('C', 'tendencia')
dot.node('D', 'Acercamiento')
dot.node('E', 'Límite de una función en un punto')
dot.node('F', '')
dot.node('G', '')
dot.node('H', '')
dot.node('I', '')
dot.node('J', '')

dot.edges(['AB'])
dot.edges(['AC'])
dot.edges(['CD'])
dot.edges(['DJ'])
dot.edges(['EF'])
dot.edges(['FG'])
dot.edges(['GH'])
dot.edges(['HI'])
dot.edges(['IJ'])

# Guardar el grafo como un archivo de imagen
dot.render(filename=main_concept, format='png', cleanup=True, \
           directory='/home/marco/Documentos/docencia/calculo/images/2_limites')

dot.view()


'/home/marco/Documentos/docencia/calculo/images/2_limites/Límites de funciones.pdf'

/snap/core20/current/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /lib/x86_64-linux-gnu/libproxy.so.1)
Failed to load module: /home/marco/snap/code/common/.cache/gio-modules/libgiolibproxy.so

(process:9987): Gtk-WARNING **: 13:39:56.239: Locale not supported by C library.
	Using the fallback 'C' locale.
/home/marco/snap/code/common/.cache/gio-modules/libgiolibproxy.so: cannot open shared object file: Permission denied
Failed to load module: /home/marco/snap/code/common/.cache/gio-modules/libgiolibproxy.so

(evince:9987): Gtk-WARNING **: 13:39:56.295: Theme parsing error: gtk-keys.css:1:0: Failed to import: Error opening file /snap/code/158/usr/share/themes/Default/gtk-3.0/gtk-keys.css: Permission denied
Gtk-Message: 13:39:56.296: Failed to load module "canberra-gtk-module"
Gtk-Message: 13:39:56.297: Failed to load module "canberra-gtk-module"
/home/marco/snap/code/common/.cache/gio-modules/libdconfsettings.so: cannot open shared object file: Per

# Ejemplo ilustrativo desde una situación problemática aplicada a la administración de empresas 16.20  (Ejemplo tomado de la Página 211 de Lina Grajales)



Un fabricante puede producir adobes para construcción en Caucasia a un costo de  $\$ 1200$ COP la unidad.  
Se estima que si los adobes se venden a un precio de $x$ pesos cada uno, los consumidores comprarán $10000 − x$ de estos al mes.   

* Exprese el beneficio de la compañía como una función del precio unitario $B = B(x)$, 
* Identifique o diga que tipo de función es el beneficio. 
* Determine la representación tabular de la función de beneficio. 
* dibuje la gráfica de esta función y utilícela para 
* estimar el precio óptimo de venta, 
* el beneficio óptimo, y 
* determine el dominio y 
* recorrido de esta función.
* Determine los siguientes límites que te ayudará a describir mejor el comportamiento económico de la función de beneficio $B = B(x)$. 

In [9]:
# Ojo, recuerde graficar con plotly para facilitar la lectura de gráficos 
# y el establecimiento de tencencias de la función a partir del gráfico de la misma
import numpy as np 
import pandas as pd 
import plotly.express as px
from sympy import * 
x, I, C, B = var('x, I, C, B')

def I(x):
    precio_de_venta = x 
    n_adobes_que_vende_en_el_mes = 10_000 - x
    ingreso = precio_de_venta*n_adobes_que_vende_en_el_mes 
    return ingreso 

def C(x): 
    costo_por_unidad = 1200
    n_adobes_que_venda_por_mes = 10_000-x 
    costo_por_mes = costo_por_unidad*n_adobes_que_venda_por_mes
    return costo_por_mes 

def B(x):
    return I(x) - C(x)

B(x)

x*(10000 - x) + 1200*x - 12000000

Identificación de la función de beneficio:  Esta función de beneficio se ve que es una función cuadrática, pues se puede llevar a la forma general $$y = ax^{2} + bx + c$$

In [16]:

precios_unitario = np.array([1200, 2200, 3200, 4200, 5200, 5300, 5400, 5500, 5600, 5700, 6200, 7200, 8200, 9200, 10_000])
beneficios_por_mes = B(precios_unitario)  

tabla = pd.DataFrame({'Precios unitarios posibles':precios_unitario,\
                    'Beneficios por mes(dados en millones)':beneficios_por_mes/1_000_000})
tabla


Precios unitarios posibles  Beneficios por mes(dados en millones)
0                         1200                                   0.00
1                         2200                                   7.80
2                         3200                                  13.60
3                         4200                                  17.40
4                         5200                                  19.20
5                         5300                                  19.27
6                         5400                                  19.32
7                         5500                                  19.35
8                         5600                                  19.36
9                         5700                                  19.35
10                        6200                                  19.00
11                        7200                                  16.80
12                        8200                                  12.60
13                        9200                                   6.40
14                       10000                                   0.00

In [13]:
solve(Eq(B(x), 0))

[1200, 10000]

In [15]:
punto_medio = (1200 + 10_000)/2
punto_medio

5600.0

Conclusión: 
* El precio optimo es 5600 pesos colombianos. 
* El beneficio máximo u optimo es de 19.36 millones de pesos


In [17]:


#fig = px.line(tabla,x = tabla.keys()[0], y = tabla.keys()[1], title=r'Relación entre $x$ y $y$')

fig = px.line(tabla,x = tabla.keys()[0], y = tabla.keys()[1], \
                 title=r'Relación entre $x$ y $B$')

fig.show()

#fig.write_html('relacion_entre_x_e_y.html', auto_open=True)

## Inferecias de la lectura del gráfico interactivo  

$$ \lim_{x \to 1200^{+}} B(x) = 0 $$

$$ \lim_{x \to 10000^{-}} B(x) = 0 $$

El dominio de la función de beneficio es $[1200, 10000]$
El rango o recorrido es el intervalo $[0, 19.36]$ millones de pesos. 


[Video de apoyo](https://www.youtube.com/watch?v=t3hmq_LweZc)

## Ejemplo ilustrativo de determinación de tendencia o límuute de una fuunción en un punto

1. Análisis de manera tabular
2. Análisis de manera gráfica. 
3. Análisis de manera algebraica y utilizando teoremas de límites. 


## Ejemplo 26.5 de la página 288 de Lina Grajales  

Encuentre $$ \lim_{x \to -2} 8x^{2} - 7x + 2 $$

### Solución

1. haciendo la tabla con pandas y luego escribiendo la respuesta o conclusión a partir de la lectura de la tabla
2. haciendo el gráfico con plotly y luego escribiendo la respuesta o conclusión a partir de la lectura del gráfico. 
3. aplicando los teoremas de límites que Lina grajales establece la página 287 (Teoremas sobre álgebra de límites).





[Video de apoyo a la determinación de límites de funciones con esta metodología]()

## Ejemplo iluatrativo



$$ f(x) =  \begin{cases} \frac{3x^{2} - 5x - 2}{x - 2} & x \neq 2 \\ 3  & x = 2\end{cases} $$

Encuentre 

$$ \lim_{x \to 2} f(x)  $$

Si $\lim_{x \to c} f(x) = L$, entonces $f(c) = L$. Esta implicación es falsa, y un contraejemplo es el anterior. 

## 1. voy a determinar el límite de manera tabular. 

In [5]:
import pandas as pd 
import numpy as np 

x = np.arange(1.9, 2.1 + 0.01, 0.015)
def f(x):
    return (3*x**2 - 5*x - 2)/(x-2)
fx = f(x) 
tabla = pd.DataFrame({'x':x, 'f(x)':fx})
tabla 


x   f(x)
0   1.900  6.700
1   1.915  6.745
2   1.930  6.790
3   1.945  6.835
4   1.960  6.880
5   1.975  6.925
6   1.990  6.970
7   2.005  7.015
8   2.020  7.060
9   2.035  7.105
10  2.050  7.150
11  2.065  7.195
12  2.080  7.240
13  2.095  7.285

## Conclusión 

$$ \lim_{x \to 2^{-}} f(x) = 7 $$
$$ \lim_{x \to 2^{+}} f(x) = 7 $$

 $$ \lim_{x \to 2} f(x) = 7 $$ 

## 2. Voy a determinar el límite de manera gráfica

In [34]:
import numpy as np 
import pandas as pd 
import pandas as pd
from plotly import express as px 

# Crear DataFrame con las coordenadas del punto rojo
punto_rojo = pd.DataFrame({'x': [2], 'y': [7]})

# Crear el gráfico de dispersión con un punto rojo grueso en (2,7)
fig = px.scatter(punto_rojo, x='x', y='y', color='x', size=[10], \
                 title=r'Análisis de tendencia de $f(x)$ en $x = 2$')  
# size controla el tamaño del marcador

# Obtener los datos de la línea y agregarlos al gráfico de dispersión
line_data = px.line(tabla, x='x', y='f(x)', \
                    ).data[0]
fig.add_trace(line_data)

# Guardar el gráfico como un archivo JPEG en un path específico
fig.write_image("/home/marco/Documentos/docencia/calculo/images/2_limites/grafico_de_la_tendencia_de_f_en_x_igual_a_2.jpeg")


# Mostrar el gráfico
fig.show()



In [25]:
punto_limite = pd.DataFrame(np.array([[2,7]]), columns=['abscisa', 'ordenada'])
punto_limite 

abscisa  ordenada
0        2         7

## Conclusión o inferencia del gráfico  



## 3. Voy a determinar el límite de esta función de manera analítica: 



$$ \lim_{x \to 2} f(x) = \lim_{x \to 2} \frac{3x^{2} - 5x - 2}{x - 2} $$

$$ = \lim_{x \to 2} \frac{(x-2)(3x +1)}{x-2} $$

$$ = \lim_{x \to 2} \frac{(3x +1)}{} $$

$$= 3(2) + 1 = 7$$

## Conclusión: 

* A medida que $x$ tiende o se aproxima a 2, $f(x)$ se aproxima a 7. 

# Próxima semana Parcial sobre límites. 

# Vocabulary 
1.  


# Visualizaciones tabulares y gráficas

## Visualización gráfica con plotly

In [ ]:
import pandas as pd 
import plotly.express as px
X=[]
Y=[]
tabla = pd.DataFrame({'anchos del cuadrado':X, 'Ordenadas':Y})



#fig = px.line(tabla,x = tabla.keys()[0], y = tabla.keys()[1], title=r'Relación entre $x$ y $y$')

fig = px.scatter(tabla,x = tabla.keys()[0], y = tabla.keys()[1], title=r'Relación entre $x$ y $y$')

fig.show()

#fig.write_html('relacion_entre_x_e_y.html', auto_open=True)

# Componente de manejo de calculadora Casio asociado al objetivo de aprendizaje de esta clase

## Descripción de la calculadora Casio  



## Tareas para realizar antes de la clase

1. Ver los videos respectivos []()
2. Memorizar las definiciones []()
3. Sintetizar o resumir la información en mapas conceptuales. []()
4. Tareas para incentivar el uso de la calculadora Casio Para ingenieros y licenciados. 

# Tareas para realizar en clase

* Modelación matemática 
* Trabajo en equipos

# Tareas para realizar después de la realización de la clase

* Actividades de memorización: 
  - escucha reiterada de grabaciones de formulas y resultados científicos. 
  - relectura de notas. 
  - ejercicios de mayor complejidad.


### No olvide asignar las tareas a los estudiantes a través de Google Colab y compartiendole el cuaderno de asignación a su correo institucional con fecha de entrega. [Google Colab](https://colab.research.google.com/?hl=es)   
  

## Asignación individual de tareas semanales 

# Horarios de monitoría y asesoría

## Monitor Manuel

Martes y jueves de 8 - 12  
Miércoles 10 - 12 y de 2 a 4 PM  
Sábado: 8 - 12

## Monitor Giovanny Reales

* Lunes a viernes de 6pm a 8pm
* Miércoles de 1 pm a 8pm

## Dificultades de aprendizaje detectados durante la clase

## Estrategias propuestas para la solución de las dificultades de aprendizaje detectadas

## Grabaciones en audio para fomentar la capacidad de memorizar enunciados de teoremas y resultados científicos y la capacidad de parafrasear estos resultados. Y así desarrollar memoria muscular para los comandos comunes. 

Se dispondrá de un lugar en Vocaroo para albergar archivos de audio de fomento a la memorización, enunciación y parafraseo de los resultados científicos, propiedades, reglas, etc. 

[Vocaroo](https://vocaroo.com/)



### Comparta sus cuadernos y videos, evidencias de su aprendizaje [aquí](https://forms.office.com/Pages/ResponsePage.aspx?id=IefhmYRxjkmK_7KtTlPBwkanXIs1i1FEujpsZgO6dXpUMEMwTEZVMzEwVVBPTElWNVg5OVEyWUhMUy4u)

## Referentes 

* [Lina Grajales: Matemáticas I para ciencias económicas](https://github.com/marco-canas/calculo/blob/main/referents/grajales/GrajalesLina_2015_MatemticasCienciasEconmicas.pdf)  
  
* [stewart precálculo](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/ERHIq62I6qFNrmxy6LZb8ZMBci7kUsyNME1nIh9yCBMJ_w?e=0zMSla)  

* [stewart cálculo](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/EZgXZjAp8QxPqOAim2hs6LcBNPLGjSHf-xwYnUVYkwa04w?e=RZdTCy)  

* [larson](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/ES71ChFeO9ZDhW3TwC5Ijk8BjxUK3Pdqz_fjHxTTFAfIAg?e=VDEjfu)

* [uzcategui](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/ETDikm-lVl1Or8XoEo9oyh0BEti9Zs8le-f0D-dBdtZmbA?e=bBsoyQ)

* [Cálculo de Purcell](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/ES60UB4h-QFFqqRQUFmkpWcBIgoLBJeqTfZjNajWNWSeJA?e=9NxjKJ)

* [Recomendación de la UNESCO sobre ciencia abierta](https://unesdoc.unesco.org/ark:/48223/pf0000379949_spa)

* [Geron, Aurelien. Hands on Machine Learning](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/Ecet27yjQzZIlT1Y_Bc2erkBUhbeYuIe6HG8i1FYZRlZww?e=gaOjqk)

* [McKinney, West. Python for data Analysis.](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/EVbi5JIeBl9ErbiUnZfGe8YBhNTnZ8sxTK5hjIOPK4UpGw?e=tfkShe)

* [ChatGPT](https://openai.com/blog/chatgpt/) 

* [Fórmulas más comunes en latex](https://wiki.geogebra.org/es/C%C3%B3digo_LaTeX_para_las_f%C3%B3rmulas_m%C3%A1s_comunes) 

### [Evaluamos al profesor Marco Cañas Aquí](https://forms.office.com/Pages/ResponsePage.aspx?id=IefhmYRxjkmK_7KtTlPBwkanXIs1i1FEujpsZgO6dXpUREJPV1kxUk1JV1ozTFJIQVNIQjY5WEY3US4u)

## [Evaluación luego de alcanzar estos objetivos de aprendizaje]()

### Continue su aprendizaje en la siguiente clase a través del siguiente [vínculo]()

# Conjeturas pedagógicas fruto de la aplicación del modelo de aprendizaje invertido y del enfoque hacia la ciencia de datos con python

1. Todo cálculo o resultado debe ser interpretado en una línea markdown del cuaderno Jupyter, inmediatamente después de la enunciación del resultado y después de la presentación de una tabla o gráfico bidimensional, de tal menera que el estudiante explicite la comprensión verbal del resultado y las inferencias o estrategias que este resultado le sugieren.  

2. Debo colocar las conjeturas pedagógicas, que el uso en el aula de [ChatGPT](https://openai.com/blog/chatgpt/) traiga para la enseñanza y el aprendizaje de las matemáticas. 

3. Promover ciencia abierta en clase de matemáticas puede ser pidiendo a los estudiantes, tomar nota en clase pero en cuadernos Jupyter y entregar siempre al final de cada clase su cuaderno con todas las notas y experimentaciones de clase replicadas. 

## Agradecimientos  

Doy gracias a Dios por la vida de mi Hijo Joseph Cañas Osorio y la madurez que ha alcanzado.

Y a mi esposa Yasmira por su apoyo, orientación y acompañamiento. 